.github/workflows/integration.yaml

```diff
@@ -69,6 +69,36 @@ jobs:
         env:
           KUBECONFIG: /home/runner/.kube/config
 
+  observability-integration:
```

Comment:
Just a thought. For consistency should we have `integration-observability` instead?
It will create a pattern along with `integration-charm-deployment`.

Reply:
+1, will change it